### DISCUSSION TITLE
Consider OpenAlex as alternative to CrossRef

### DISCUSSION THREAD

**Discussion by Adafede** [lotusnprod/lotus-processor#22] [Ideas]
Following multiple discussions on WikiCite Telegram, consider moving from CrossRef to OpenAlex in the future.

@egonw @Daniel-Mietchen any comments? Too soon?

**Comment by egonw** [comment]
Neither too soon, nor too later. OpenAlex is not a replacement for CrossRef; it pulls in information from CrossRef, which is the authoritative source here. Now, OpenAlex does have *more* information than CrossRef, and may contain corrections to publisher data.

So, for what step are you considering it?

**Reply by Adafede** [reply]
Ok...I had the feeling they do not simply _add_ a layer of information but also improve standardization. I felt upon multiple badly formatted titles in CrossRef and thought OpenAlex might help, I was probably wrong...

**Reply by egonw** [reply]
They might, but curation is hard and expensive. Just ask Heather and Jason :)
